<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Panel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="greeting">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hello,</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>[USERNAME]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="statButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cashierButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Employees</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Inventory</string>
     </property>
    </widget>
    <widget class="QPushButton" name="promotionButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>500</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Promotions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Logs</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>logout</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>711</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Icon</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="iconTable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>371</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="image">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>250</width>
       <height>250</height>
      </rect>
     </property>
     <property name="text">
      <string>                   [IMAGE HERE]</string>
     </property>
    </widget>
    <widget class="QLabel" name="iconNamePreview">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[ICON NAME]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="returnButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorMsg">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>360</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[INVISIBLE ERROR MSG &amp; BUTTON]</string>
     </property>
    </widget>
    <widget class="QPushButton" name="overwriteButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>407</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Overwrite Icon</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note: [IMAGE HERE] must be 250x250</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
